dockerfiles/e2e-saas/Dockerfile

```diff
@@ -4,7 +4,7 @@
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-v10.html
 
-ARG TAG=latest
+ARG TAG=7.10.0
```

Comment:
@ScrewTSW Why did you changed this argument setting? If you have no strong reason, please revert this change. This argument can be changed using `docker run` command.

Reply:
It's just for testing right now. in 7.10.0 PR the `latest` tag is scuffed, because the latest bits are 7.11.0 if I'm correct.

Reply:
This is not true. PR check is using the version that is in a pom.xml [1]. So please revert those changes. Also, changes that should not be merged should have a clear message, not be hidden in some commit.
[1] https://github.com/redhat-developer/rh-che/blob/master/functional-tests/devscripts/run_tests.sh#L95

Reply:
(and just a note - latest is currently 7.10.0 ;) )

Reply:
fixed, thank you for the clarification